virtool/models.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, String
+
+Base = declarative_base()
+
+
+class Label(Base):
+    __tablename__ = 'labels'
+
+    id = Column(String, primary_key=True)
+    name = Column(String)
+    color = Column(String)
```

Comment:
Since all colors are exact six characters, you can set a length of 6 on the string.

https://docs.sqlalchemy.org/en/13/core/type_basics.html#sqlalchemy.types.String